borgmatic/hooks/mysql.py

```diff
@@ -206,9 +207,9 @@ def restore_data_source_dump(
     password = connection_params['password'] or data_source.get(
         'restore_password', data_source.get('password')
     )
-
+    mysql_restore_command = data_source.get('mysql_command') or 'mysql'
     restore_command = (
-        ('mysql', '--batch')
+        (mysql_restore_command, '--batch')
```

Comment:
If I'm reading this correctly, you're missing one other instance of `mysql`/`mysqldump` in `mysql.py`: Around line 39 in `database_names_to_dump()`.